wrapper-cpp/modules/c_meeting_participants_interface.cpp

```
@@ -69,6 +69,22 @@ extern "C" bool is_host(ZOOMSDK::IUserInfo *user_info) {
     return user_info->IsHost();
 }
 
+extern "C" const zchar_t* meeting_participants_get_avatar_path(ZOOMSDK::IUserInfo *user_info) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)